crypto/src/conversation/handshake.rs

```diff
@@ -52,41 +52,53 @@ impl MlsProposalBundle {
 /// Creating proposals
 impl MlsConversation {
     /// see [openmls::group::MlsGroup::propose_add_member]
+    #[crate::durable]
```

Comment:
Maybe we switch all the `#[crate::durable]` to `#[cfg_attr(test, crate::durable)]` ?
I don't think we don't want to inflate codesize with test utils